src/microbe_stage/organelle_components/ExternallyPositionedComponent.cs

```diff
@@ -21,7 +26,7 @@ public void OnAttachToCell(PlacedOrganelle organelle)
     {
         this.organelle = organelle;
         organellePos = Hex.AxialToCartesian(organelle.Position);
-
+        defaultVisualPos = Vector3.Forward;
```

Comment:
Assign a pos with a "Forward" is a little bit strange. Maybe add a comment or use new instead?

Reply:
No reason to use new, vector forward already exists creating a new vector takes more time.